src/pages/docs/tutorials/tutorial-kitten-browser/lists/index.md

```diff
@@ -109,6 +109,15 @@ computed: {
 	}
 },
 ```
+**Note**: We could do the same computation using the Repeater's default `indexProp` (`data-index`) by changing the destructuring to get the value out of the Kitten instance's data attributes (DOM attributes prepended with `data-`) instead. Component props will include all DOM data attributes as `dataset`, so the value of the DOM attribute `data-index` would be found in `props.dataset.index` and `data-foo`'s value would be in `props.dataset.foo`.
+```js
+computed: {
+	url: ({dataset: {index}, size}) => {
```

Comment:
I'm not sure this is true. The DOM node with have a `dataset` member but that won't exist in React's vdom. We could destructure `({'data-index': index, size})` but I don't think `dataset` would work. Though I've never tried this so I could be wrong!

Reply:
Good catch!  Updated.